scripts/template-updater.ts

```diff
@@ -1,64 +1,39 @@
+#!/usr/bin/env node
 import fs from 'fs';
 import path from 'path';
 import semver from 'semver';
 import { err, JSONable, log } from './lib-concourse';
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
-
-function readFile(filePath: string): string {
-    log(`reading content from file: ${filePath}`);
-    if (fs.existsSync(filePath)) {
-        const stat = fs.statSync(filePath);
-        if (stat.isFile()) {
-            log(`file found: ${filePath}`);
-            const buffer = fs.readFileSync(filePath);
-            return buffer.toString();
-        } else {
-            log(`not a file: ${filePath}`);
-        }
-    } else {
-        log(`file not found: ${filePath}`);
-    }
-    log(`unable to read content from file: ${filePath}`);
-    return null;
-}
-
 function readJSONTemplate(filePath: string): JSONable {
+    let contents = '';
     try {
-        const content = readFile(filePath);
-        return content !== null && JSON.parse(content);
+        contents = fs.readFileSync(filePath).toString();
     } catch (error) {
-        err('error in parsing into JSON object: ', JSON.stringify(error));
+        err(`error in reading the JSON template: ${filePath}. error: `, JSON.stringify(error));
```

Comment:
just set `error.message = `error in reading the JSON template: ${filePath}. error:  ${error.message}` ?
Then the throw will log your custom message.

Reply:
done